### PR TITLE
libav 12

### PR DESCRIPTION
**Pull request by woodruffw**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libav.rb [hunk 1]
```diff
@@ -60,11 +60,9 @@ class Libav < Formula
   depends_on "speex" => :optional
   depends_on "theora" => :optional
 
-  # Fixes the use of a removed identifier in libvpx;
-  # will be fixed in the next release.
   patch do
-    url "https://github.com/libav/libav/commit/4d05e9392f84702e3c833efa86e84c7f1cf5f612.patch"
-    sha256 "78f02e231f3931a6630ec4293994fc6933c6a1c3d1dd501989155236843c47f9"
+    url "https://bugzilla.libav.org/attachment.cgi?id=656"
```
Comment (zmwangx) [inline Formula/libav.rb]: Probably add https://bugzilla.libav.org/show_bug.cgi?id=1033 in a comment for easier progress tracking?

By the way, https://bugzilla.libav.org/robots.txt blocks search engines. So crazy.

Comment (woodruffw) [inline Formula/libav.rb]: Good call, I'll tack that in once the build completes/explodes.

**Comment by woodruffw** [comment]
Just pushed an inlined version with @JCount's fixup of the patch, I'll replace it with the homebrew-patches version once we confirm it works.

**Comment by JCount** [comment]
🎊

**Comment by woodruffw** [comment]
:tada:

**Pull request closed without merging**